api/model/src/test/java/org/projectnessie/model/TestValidation.java

```diff
@@ -250,4 +250,42 @@ void pathRefName(String rest, String ref, String hashOnRef, String relativeSpec)
     soft.assertThat(matcher.group(2)).isEqualTo(hashOnRef);
     soft.assertThat(matcher.group(3)).isEqualTo(relativeSpec);
   }
+
+  @ParameterizedTest
+  @CsvSource({
+    ",false",
+    "'',false",
```

Comment:
Huh? IIRC quotes should not be allowed here.

Reply:
This is testing an empty string. The line above is testing `null`.

Reply:
Oh, right - CSV :facepalm: